content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -210,10 +210,22 @@ The initial imported plugins file can be
 
 ### Support file
 
-By default Cypress will automatically include the support file
-`cypress/support/index.js`. This file runs **before** every single spec file. We
-do this purely as a convenience mechanism so you don't have to import this file
-in every single one of your spec files.
+By default Cypress will automatically include the supportFile. This file runs
+**before** every single spec file. We do this purely as a convenience mechanism
+so you don't have to import this file in every single one of your spec files.
+
+<Alert type="info">
+
+<strong class="alert-header"><Icon name="cogs"></Icon> Support File Per Test
```

Comment:
Do we use the `cogs` icon elsewhere? Either way, I really like the idea of having a general callout for testing-type specific options, so it would be good to make this into a directive. In it, this code:

```
::testing-type-specific-option{option=supportFile}
```

Could generate an `info` callout like this:

> ### ⚙️ Testing Type-Specific Option
> 
> Depending on which testing type you are using, you may configure the `supportFile` option accordingly.
> 
> - [e2e](/guides/references/configuration#e2e) testing type specific options
> - [component](/guides/references/configuration#component) testing type specific options

Reply:
Looks like we only use the cog again here https://docs.cypress.io/guides/getting-started/testing-your-app#Step-3-Configure-Cypress. Can take it or leave it. Thought it was a helpful visual to a fairly important callout here.

Reply:
You've missed the point of the `::testing-type-specific-option` directive. It wasn't to replicate the `::include` functionality we already have, it was to take this content (probably stored inside the directive .js file in a template string literal) and replace the `OPTION_HERE` bit with whatever was specified as the `option` attribute:

> ### ⚙️ Testing Type-Specific Option
> 
> Depending on which testing type you are using, you may configure the `OPTION_HERE` option accordingly.
> 
> - [e2e](/guides/references/configuration#e2e) testing type specific options
> - [component](/guides/references/configuration#component) testing type specific options

So that `::testing-type-specific-option{option=supportFile}` would render:

> ### ⚙️ Testing Type-Specific Option
> 
> Depending on which testing type you are using, you may configure the `supportFile` option accordingly.
> 
> - [e2e](/guides/references/configuration#e2e) testing type specific options
> - [component](/guides/references/configuration#component) testing type specific options

But  `::testing-type-specific-option{option=specPattern}` would render:

> ### ⚙️ Testing Type-Specific Option
> 
> Depending on which testing type you are using, you may configure the `specPattern` option accordingly.
> 
> - [e2e](/guides/references/configuration#e2e) testing type specific options
> - [component](/guides/references/configuration#component) testing type specific options